docs/rst/installation/linux_installation.rst

```diff
@@ -0,0 +1,249 @@
+
+
+.. _colcon_installation_linux:
+
+Colcon installation
+-------------------
+
+colcon_ is a command line tool based on CMake_ aimed at building sets of software packages.
+This section explains how to use it to compile *eProsima Fast DDS Statistics Backend* and its dependencies.
+
+#. Install the ROS 2 development tools (colcon_ and vcstool_) by executing the following command:
+
+   .. code-block:: bash
+
+       pip3 install -U colcon-common-extensions vcstool
+
+   .. note::
+
+       If this fails due to an Environment Error, add the :code:`--user` flag to the :code:`pip3` installation command.
+
+#. Create a :code:`Fast-DDS-statistics-backend` directory and download the `repos` file that will be used to install
+   *eProsima Fast DDS Statistics Backend* and its dependencies:
+
+   .. |wget-statistics-backend| replace:: wget https://raw.githubusercontent.com/eProsima/Fast-DDS-statistics-backend/master/fastdds_statistics_backend.repos
+
+   .. code-block:: bash
+
+       mkdir ~/Fast-DDS-statistics-backend
+       cd ~/Fast-DDS-statistics-backend
+       |wget-statistics-backend|
+       mkdir src
+       vcs import src < fastdds_statistics_backend.repos
+
+#. Build the packages:
+
+   .. code-block:: bash
+
+       colcon build
+
+.. note::
+
+    Being based on CMake_, it is possible to pass the CMake configuration options to the :code:`colcon build`
+    command. For more information on the specific syntax, please refer to the
+    `CMake specific arguments <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_
+    page of the colcon_ manual.
+
+.. _run_app_colcon_sl:
+
+Run an application
+^^^^^^^^^^^^^^^^^^
+
+When running an instance of an application using *eProsima Fast DDS Statistics Backend*,
+the colcon overlay built in the dedicated :code:`Fast-DDS-statistics-backend` directory
+must be sourced.
+There are two possibilities:
+
+* Every time a new shell is opened, prepare the environment locally by typing the
+  command:
+
+  .. code-block:: bash
+
+      source ~/Fast-DDS-statistics-backend/install/setup.bash
+
+* Add the sourcing of the colcon overlay permanently to the :code:`PATH`, by typing the following:
+
+  .. code-block:: bash
+
+      echo 'source ~/Fast-DDS-statistics-backend/install/setup.bash' >> ~/.bashrc
+
+
+.. _cmake_installation_linux:
+
+CMake installation
+------------------
+
+This section explains how to compile *eProsima Fast DDS Statistics Backend* with CMake_,
+either :ref:`locally <local_installation_sl>` or :ref:`globally <global_installation_sl>`.
+
+.. _local_installation_sl:
+
+Local installation
+^^^^^^^^^^^^^^^^^^
+
+#. Follow the `*eProsima Fast DDS* local installation guide<https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#local-installation>`
+   to install *eProsmia Fast DDS* and all its dependencies
+
+#. Install *eProsima Fast DDS Statistics Backend*:
+
+   .. code-block:: bash
+
+       cd ~/Fast-DDS
```

Comment:
I think that naming the root directory `Fast-DDS` could lead to confusion... I am not sure but maybe it is better to call it `Fast-DDS-workspace`.

Reply:
The directory name was taken from the previous step of installing Fast DDS, which already uses that name. I find it preferable to keep the whole workspace (Fast DDS and Statistics Backend) in the same place.